### PR TITLE
Add docs for mocking command

### PR DESCRIPTION
**Pull request by kylehodgetts**

### CODE REVIEW

File: docs/cli/mock-cmd.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                order: 13
+                url: "http://mockedURL.com"
+```
+
+It has a single path with 3 content types.
+- `application/json` specifies a response schema which kusk will generate a generated response that matches
+- `applications/xml` and `text/plain` specifies examples which kusk will return as is.
+
+### Launch Kusk Mocking Server
+```shell
```
Comment (jasmingacic) [inline docs/cli/mock-cmd.md:64]: I was thinking if we had a GIF here showing how it works. It would look very nice

**Review by jasmingacic — DISMISSED** [review]
Looks good to me.

**Review by povilasv — APPROVED** [review]

File: docs/cli/mock-cmd.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+⏳ watching for file changes in todo-backend-api.yaml 
+```
+
+The mock server is now running and will watch for any changes you make to fake todo-backend-api.yaml.
+
+### Interacting with your API
+Let's curl the endpoint for a JSON response
+
+```shell
+➜ curl -H "Accept: application/json" localhost:8080/todos | jq
```
Comment (mbana) [inline docs/cli/mock-cmd.md:79]: Minor points, please use standard convention of `$` instead of `➜`.

**Review by mbana — APPROVED** [review]

File: docs/cli/mock-cmd.md [hunk 3]
```diff
@@ -61,6 +61,8 @@ It has a single path with 3 content types.
 - `applications/xml` and `text/plain` specifies examples which kusk will return as is.
 
 ### Launch Kusk Mocking Server
+![kusk mocking example](../img/kusk-mock.gif)
```
Comment (jasmingacic) [inline docs/cli/mock-cmd.md:64]: nice, really nice

**Review by jasmingacic — APPROVED** [review]

**Pull request merged**